pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -122,15 +133,11 @@ private void removeRateLimiter(String tableNameWithType) {
 
   /**
    * Get QuotaConfig from property store.
-   * @param rawTableName table name without table type.
-   * @param tableType table type: offline or real-time.
+   * @param tableNameWithType table name with table type.
    * @return QuotaConfig, which could be null.
    */
-  private QuotaConfig getQuotaConfigFromPropertyStore(String rawTableName, CommonConstants.Helix.TableType tableType) {
-    ZkHelixPropertyStore<ZNRecord> propertyStore = _helixManager.getHelixPropertyStore();
-
-    String tableNameWithType = TableNameBuilder.forType(tableType).tableNameWithType(rawTableName);
-    TableConfig tableConfig = ZKMetadataProvider.getTableConfig(propertyStore, tableNameWithType);
+  private QuotaConfig getQuotaConfigFromPropertyStore(String tableNameWithType) {
+    TableConfig tableConfig = ZKMetadataProvider.getTableConfig(_propertyStore, tableNameWithType);
```

Comment:
I remember some discussions from before that the property store handle is stale if helix loses zk connection. Has that assumption changed (or is my recollection wrong)? Helix code seems to be resetting the handle on disconnect() as well.

Reply:
Even when the zk connection is lost, some exceptions will be thrown. In that case, nothing change in this class. So it'd be fine here.